src/models/StyledComponent.js

```diff
@@ -250,7 +246,7 @@ export default (ComponentStyle: Function, constructWithOptions: Function) => {
 
         const newComponentId =
           previousComponentId &&
-          `${previousComponentId}-${isTag(tag) ? tag : getComponentName(tag)}`
```

Comment:
The tag here doesn’t need escaping :)

Reply:
Same question - why does this one not need to be escaped? Seems like whenever we are dealing with `displayName`, we should always escape to be safe?

Reply:
Well the component name here will need escaping but if isTag is true then it’s just an element name